tests/integration/test_tls.py

```diff
@@ -110,7 +110,7 @@ async def test_scale_up_tls(ops_test: OpsTest):
 async def test_client_relate_maintains_quorum(ops_test: OpsTest):
     dummy_name = "app"
     app_charm = await ops_test.build_charm("tests/integration/app-charm")
-    await ops_test.model.deploy(app_charm, application_name=dummy_name, num_units=1),
+    await ops_test.model.deploy(app_charm, application_name=dummy_name, num_units=1)
```

Comment:
Is this change related to the fixes that were added through this PR? Something in that way would be interesting.

Reply:
It's just fixes. I've got a larger ticket on the way for somewhat standardising and extending all the unit/int testing, so thought I'd leave that for a separate PR